sceptre/cli/launch.py

```diff
@@ -24,9 +24,16 @@
     is_flag=True,
     help="If set, will delete all stacks in the command path marked as obsolete.",
 )
+@click.option(
+    "--disable-rollback/--enable-rollback",
+    default=False,
```

Comment:
Unless I'm missing something, I think we might want `default=None` here. That way if the user doesn't specify `--disable...` or `--enable...` it will fall back to whatever's in the StackConfig (or False if nothing's there).

Reply:
Great catch!

Reply:
Is this something for @zaro0508 to deal with or should I open another PR?

Reply:
If you have opportunity and time to get it done, I say go for it. I'm not sure what @zaro0508 has going on over the next few days and if he'll have time. (I'll let him speak for himself, though).

Reply:
yes, please. would love to see a PR from you @dboitnot 😄

Reply:
Added #1288. Since it was a one-word update I tried doing it through the GitHub interface.